src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,1996 @@
+            {
+                string recognizedString = Marshal.PtrToStringAnsi(recognizedText);
+                string msgString = Marshal.PtrToStringAnsi(msg);
+
+                if (_allRecognitionResult != null)
+                {
+                    AllRecognitionResultEventArgs args = new AllRecognitionResultEventArgs(result, recognizedString, msgString);
+                    _allRecognitionResult?.Invoke(null, args);
+                    if (result == RecognizedResult.Rejected)
+                        return true;
+                }
+                
+                if (result == RecognizedResult.Success)
+                {
+                    VoiceCommandsGroup cmdList = new VoiceCommandsGroup(new SafeCommandListHandle(vcCmdList));
+                    IEnumerable<VoiceCommand> selectedList = _recognizedCommandsDelegate.Invoke(cmdList.Commands, recognizedString, msgString);
+                    if (selectedList == null)
+                        return true;
+
+                    VoiceCommandsGroup resultList = new VoiceCommandsGroup();
+                    foreach (VoiceCommand command in selectedList)
+                    {
+                        resultList.Commands.Add(command);
+                    }
+                    error = VcMgrSetSelectedResults(resultList._handle);
+                    if (error != ErrorCode.None)
+                    {
+                        Log.Error(LogTag, "SetSelectedResults Failed with error " + error);
+                        throw ExceptionFactory.CreateException(error);
+                    }
+                    return false;
+                }
+                return true;
+            };
+
+            if (_allRecognitionResult == null && _recognizedCommandsDelegate == null)
+            {
+                error = VcMgrSetAllResultCb(_recognizedCommandsSelectionDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add SetRecognizedCommandsSelectionDelegate Failed with error " + error);
+                    throw ExceptionFactory.CreateException(error);
+                }
+            }
+            _recognizedCommandsDelegate = recognizedCommandsDelegate;
+        }
+
+        /// <summary>
+        /// Called when client gets the all recognition results from vc-daemon.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<AllRecognitionResultEventArgs> AllRecognitionResult
```

Comment:
```suggestion
        public static event EventHandler<AllRecognitionResultEventArgs> AllRecognitionResultDelivered
```
```suggestion
        public static event EventHandler<AllRecognitionResultEventArgs> AllRecognitionResultReceived
```

Reply:
I will change to 'AllRecognitionResultReceived'